test/unittests/libethashseal/EthashTest.cpp

```diff
@@ -55,7 +55,8 @@ BOOST_AUTO_TEST_CASE(calculateDifficultyByzantiumWithoutUncles)
     header.setNumber(0x2001);
     header.setTimestamp(130);
 
-    BOOST_CHECK_EQUAL(ethash.calculateDifficulty(header, parentHeader), 999024);
```

Comment:
I'd leave this without a change, the tests here check `Ethash` class, we'd like to be sure that it calls the correct calculation

Reply:
how?
we've just removed calculateDifficulty from ethash into a separate function

Reply:
the test doesnt fail. that should be enough. it actually tests that function work as it was before

Reply:
Ah right, for a moment it seemed to me that `Ethash::calculateDifficulty` still existed. Ok then, this is fine